test/gov/agbot_del_loop.sh

```diff
@@ -4,24 +4,43 @@
 
 for (( ; ; ))
 do
+   if [ "${EXCH_APP_HOST}" = "http://exchange-api:8080/v1" ]; then
+     AGID1=$(curl -sS http://localhost:81/agreement | jq -r '.agreements.active[0].current_agreement_id')
+     AGID2=$(curl -sS http://localhost:81/agreement | jq -r '.agreements.active[1].current_agreement_id')
+     AGID3=$(curl -sS http://localhost:81/agreement | jq -r '.agreements.active[2].current_agreement_id')
+     AGID4=$(curl -sS http://localhost:81/agreement | jq -r '.agreements.active[3].current_agreement_id')
 
-   AGID1=$(curl -sS http://localhost:81/agreement | jq -r '.agreements.active[0].current_agreement_id')
-   AGID2=$(curl -sS http://localhost:81/agreement | jq -r '.agreements.active[1].current_agreement_id')
-   AGID3=$(curl -sS http://localhost:81/agreement | jq -r '.agreements.active[2].current_agreement_id')
-   AGID4=$(curl -sS http://localhost:81/agreement | jq -r '.agreements.active[3].current_agreement_id')
+     echo "Agbot deleting agreements"
+     echo "Deleting $AGID1"
+     DEL=$(curl -sS -X DELETE http://localhost:81/agreement/$AGID1)
 
-   echo "Agbot deleting agreements"
-   echo "Deleting $AGID1"
-   DEL=$(curl -sS -X DELETE http://localhost:81/agreement/$AGID1)
+     echo "Deleting $AGID2"
+     DEL=$(curl -sS -X DELETE http://localhost:81/agreement/$AGID2)
 
-   echo "Deleting $AGID2"
-   DEL=$(curl -sS -X DELETE http://localhost:81/agreement/$AGID2)
+     echo "Deleting $AGID3"
+     DEL=$(curl -sS -X DELETE http://localhost:81/agreement/$AGID3)
 
-   echo "Deleting $AGID3"
-   DEL=$(curl -sS -X DELETE http://localhost:81/agreement/$AGID3)
+     echo "Deleting $AGID4"
+     DEL=$(curl -sS -X DELETE http://localhost:81/agreement/$AGID4)
+   else
+     AGID1=$(kubectl -n kube-system exec -ti $(kubectl -n kube-system get pods | grep -v 'agbot-db' | grep -m 1 'edge-computing-agbot' |  awk '{print $1}') -- sh -c "export ANAX_PORT=\$(cat /etc/horizon/anax.json | jq -r .AgreementBot.APIListen | awk -F: '{print \$NF}'); curl http://localhost:\$ANAX_PORT/agreement | jq -r '.agreements.active[0].current_agreement_id'")
```

Comment:
I thought this entire path was all going to be commented out until we did more work? I dont think this will work for anyone else who tries to run this remotely?

Reply:
It does not work, but also does not affect the run. It will give the user an unauthorized message, but that is the extent of the damage.

Reply:
OK, that's a fair point.